### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Leh-02**
- [DEMO LINK](https://Leh-02.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Leh-02.github.io/layout_product-cards/report/html_report/)

- [x] there 2 BEM blocks `card` and `stars` each in it own file
- [x] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x] SCSS Variables are used for main values and placed in a **separate file**
- [x] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,11 +3,62 @@
...
+      </div>
+
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br>5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__name">
+        <div class="card__product-code">Product code: 195434</div>
+      </div>
```
Comment (IvanFesenko) [inline src/index.html:34]: do not use div for text content - use semantic tags instead (h, p etc)

File: src/index.html [hunk 2]
```diff
@@ -3,11 +3,62 @@
...
+        <div class="stars__block">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+
+        <div class="stars__review">Reviews: 5</div>
+      </div>
```
Comment (IvanFesenko) [inline src/index.html:46]: Stas should wokr with modifiers 1-5
```suggestion
      <div class="raiting">
        <div class="stars">
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
          <span class="stars__star"></span>
        </div>

        <p class="raiting__review">Reviews: 5</p>
      </div>
```

File: src/index.html [hunk 3]
```diff
@@ -3,11 +3,62 @@
...
+
+        <p class="card__priceAmount">$2,199</p>
+      </div>
+
+
+      <a
+        href="#"
+        class="card__buy"
+        data-qa="hover"
+      >buy</a>
```
Comment (IvanFesenko) [inline src/index.html:59]:
```suggestion
      <a
        href="#"
        class="card__buy"
        data-qa="hover"
      >
        buy
      </a>
```

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,119 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:4]:
```suggestion
  width: 200px;
```

File: src/styles/blocks/card.scss [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+  height: 406px;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:5]: Do not hard code height
```suggestion

```

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+  }
+
+  &__logo {
+    height: 100%;
+    width: 100%;
+  }
+
+  &__title {
+    width: 166px;
+    height: 36px;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:27]:
```suggestion

```

File: src/styles/blocks/card.scss [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+    margin-top: 40px;
+    margin-left: 16px;
+    margin-right: 16px;
+
+    color: $main-accent;
+    font-family: Roboto, sans-serif;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:33]: move into `html` selector

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+    line-height: 18px;
+  }
+
+  &__name {
+    margin-top: 4px;
+    margin-left: 16px;}
+
+  &__product-code {
+    color: $secondary;
+    font-family: Roboto, sans-serif;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:46]:
```suggestion

```

File: src/styles/blocks/card.scss [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:49]: It's default values - redundant
```suggestion

```

File: src/styles/blocks/card.scss [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+    line-height: 14px;
+  }
+
+
+  &__price{
+    display: flex;
+    width: 166px;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:56]:
```suggestion

```

File: src/styles/blocks/card.scss [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+    align-items: flex-end;
+
+    margin-left:16px;
+    margin-right:16px;
+    margin-top:24px;
+  }
+
+  &__priceText {
+    color: $secondary;
+    font-family: Roboto, sans-serif;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:67]:
```suggestion

```

File: src/styles/blocks/card.scss [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:70]:
```suggestion

```

File: src/styles/blocks/card.scss [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__priceAmount {
+    color: $main-accent;
+    text-align: right;
+    font-family: Roboto, sans-serif;
```
Comment (IvanFesenko) [inline src/styles/blocks/card.scss:78]:
```suggestion

```

File: src/styles/blocks/stars.scss [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  &__star:nth-child(-n+4){
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__review {
+    color: $main-accent;
+    text-align: right;
+    font-family: Roboto, sans-serif;
```
Comment (IvanFesenko) [inline src/styles/blocks/stars.scss:34]:
```suggestion

```

File: src/styles/blocks/stars.scss [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+  }
```
Comment (IvanFesenko) [inline src/styles/blocks/stars.scss:39]: that not beongs to stars block

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Reports page not available, Check my comments
2024-01-26_15h48_48

**Comment by Leh-02** [comment]
@IvanFesenko , can you check my new commits, please

**Pull request closed without merging**